iceoryx_posh/include/iceoryx_posh/popo/listener.hpp

```diff
@@ -76,7 +76,6 @@ enum class ListenerError
     LISTENER_FULL,
     EVENT_ALREADY_ATTACHED,
     EMPTY_EVENT_CALLBACK,
-    EMPTY_INVALIDATION_CALLBACK
```

Comment:
Can we still have an empty event callback?

Reply:
This is no longer possible. The only thing a user now can do is to provide a semi empty callback:
```cxx
cxx::function<void()> empty = []{};
```

It is a valid function with no statements. But this is still callable and not preventable.